core/templates/domain/exploration/WrittenTranslationObjectFactory.ts

```diff
@@ -20,6 +20,11 @@
 import { downgradeInjectable } from '@angular/upgrade/static';
 import { Injectable } from '@angular/core';
 
+export interface ITranslationBackendDict {
+  html: string;
+  'needs_update': boolean;
+}
```

Comment:
Maybe use the quotes consistently? So if there is at least one quoted item quote all the items. Ditto for other places.

Reply:
done